modules/perception/obstacle/camera/cipv/BUILD

```diff
@@ -0,0 +1,20 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "perception_obstacle_camera_cipv",
+    srcs = ["cipv.cpp"],
```

Comment:
cipv.cpp => cipv.cc

Reply:
Thank you for the review! I will update it and issue PR again.